src/components/common/Input/index.js

```diff
@@ -1,22 +1,38 @@
 import React from 'react';
 import { string, bool, oneOfType, array } from 'prop-types';
 import { View, TextInput, Text } from 'react-native';
+
+import common from 'constants/commonStyles';
 import styles from './styles';
 
-const Input = ({ label, invalid, error, ...props }) => (
+const Input = ({ label, invalid, error, fromSignupForm, short, ...props }) => (
```

Comment:
Instead of `fromSignupForm` what about something like `showErrorMessage`?

Reply:
Yeah, I think the same as peter